climada_petals/hazard/wildfire.py

```diff
@@ -25,24 +25,42 @@
 import itertools
 from dataclasses import dataclass
 from datetime import date
+from pathlib import Path
+import copy
 
 import numpy as np
 import matplotlib.pyplot as plt
 import pandas as pd
 from scipy import sparse
+import scipy.ndimage # added by Sam G.
 from sklearn.neighbors import BallTree
 from sklearn.cluster import DBSCAN
 import numba
+import rasterio
+import shapely.geometry
+import random # added by Sam G.
```

Comment:
In case you did not know, random numbers can also be generated with numpy or scipy depending on the needs.